.github/workflows/test.yml

```diff
@@ -16,23 +16,54 @@ jobs:
       fail-fast: false
       matrix:
         go-version: [1.15.x, 1.16.x, 1.17.x]
+        libseccomp: ["", "v2.5.2", "HEAD"]
```

Comment:
Thanks for doing this @kolyshkin, but as mentioned on another issue/PR somewhere, I'm not a big fan of testing against the default Ubuntu package as I don't want to see our CI failing because a distro made a mistake with the packaging (mistakes are our job! <g>).

I understand the desire to test against downstream distros, but I would much rather see us remain as distro-agnostic as possible and focus on the upstream bits when doing upstream testing.

Reply:
Seems like a reasonable argument to me.  I think the matrix still makes sense, but obviously it would become
`libseccomp: ["v2.5.2", "HEAD"]`

Reply:
Yes, the matrix testing is really nice, I don't want to lose that, I just don't want to bother with testing and gating on Ubuntu's package.

Reply:
I think testing against whatever a distro is providing adds value, as personally I'd like to know if it fails. Besides, what we use here in CI is a stable distro, not some bleeding edge daily build or something, and we already depend on it a lot (glibc/gcc/gperf/binutils and so on).

It seems that issue you refer to can be solved by not treating all the CI jobs equal, but rather classify them into "required" or "optional". If you and this can surely be placed into the latter category (same is true for test against git tip, which IMO have much higher chance of breakage).

Anyway, it's easy to implement; done. Please let me know if you think it makes sense or not to explicitly add 2.5.1 (compiled from source) to the matrix (I tend to think it's not).

Reply:
Ended up having various minor supported libseccomp releases, as this is what we had in travis.